src/extension_commands.ml

```diff
@@ -1274,7 +1316,9 @@ module Navigate_holes = struct
            ->
            show_message
              `Warn
-             "The installed version of `ocamllsp` does not support typed hole navigation"
+             "The installed version of `ocamllsp` does not support typed hole navigation";
+           let _ = Ocaml_lsp.suggest_to_upgrade_ocaml_lsp_server () in
```

Comment:
We're missing the same thing in the `type_selection.ml` module :-)

Reply:
Also having two messages shown is not ideal:

<img width="671" alt="image" src="https://github.com/user-attachments/assets/05084219-8d31-4523-8c3f-55fa6c8d3e37" />

Maybe we could have an option to customize the message of the `suggest_to_upgrade_ocaml_lsp_server` function ?